databind/src/databind/json/tests/converters_test.py

```diff
@@ -713,3 +713,25 @@ def of(cls, v: str) -> "MyCls":
     mapper = make_mapper([JsonConverterSupport()])
     assert mapper.serialize(MyCls(), MyCls) == "MyCls"
     assert mapper.deserialize("MyCls", MyCls) == MyCls()
+
+
+UnboundGeneric = t.TypeVar("UnboundGeneric")
+
+
+@dataclasses.dataclass
+class GenericClass(t.Generic[UnboundGeneric]):
+    a_field: int
+
+
+@dataclasses.dataclass
+class InheritGeneric(GenericClass):  # type: ignore[type-arg]
+    b_field: str
+
+
+def test_serialize_generic_dataclass() -> None:
+    """Regression test for #66: dataclasses inheriting from Generic with an uninstantiated TypeVar don't get their
+    parents' fields.
+    """
+    obj = InheritGeneric(2, "hi")
+    mapper = make_mapper([SchemaConverter(), PlainDatatypeConverter()])
+    assert mapper.serialize(obj, InheritGeneric) == {"a_field": obj.a_field, "b_field": obj.b_field}
```

Comment:
Could you add the reverse (deserialize) here as well?

Reply:
Done thanks